.github/release-drafter.yml

```
@@ -3,28 +3,28 @@ tag-template: 'v$RESOLVED_VERSION'
 categories:
   - title: '💥 Breaking Changes'
     labels:
-      - breaking
+      - 💥 breaking
   - title: '✨ Enhancements'
     labels:
       - feature
-      - enhancement
+      - ✨ enhancement
   - title: '🐛 Bug Fixes'
     labels:
       - fix
       - bugfix
-      - bug
+      - 🐛 bug
   - title: '🏗  Maintenance'
```

While we're here, do me a favor and fix my double space. 😆 

```suggestion
  - title: '🏗 Maintenance'
```